spec/hutch/message_handler_spec.rb

```diff
@@ -1,51 +1,20 @@
 require 'spec_helper'
-require 'hutch/worker'
```

Comment:
I actually deleted the worker_spec because all the tests were related to queue setup. Github decided I actually moved the file.